src/faq.md

```diff
@@ -266,6 +280,21 @@ You can also use the [Flutter framework]({{site.flutter}}), which has [web suppo
 [No.]({{site.news}}/2015/03/dart-for-entire-web.html)
 Dart is designed to compile to JavaScript to run across the modern web.
 
+### Q. Is Dart single-threaded on the web?
+
+Dart is *mostly* single-threaded on the web. 
+Web apps can use concurrency features like async-await,
```

Comment:
Async-await, futures and streams is not what I'd consider "Conccurency". Only web-workers actually allow concurrent execution.